### PR TITLE
pla_util 2.1.1

### PR DESCRIPTION
**Pull request by serock**
Initial release of pla_util crate.

### CODE REVIEW

File: index/pl/pla_util/pla_util-2.1.1.toml [hunk 1]
```diff
@@ -0,0 +1,64 @@
+#  SPDX-License-Identifier: GPL-3.0-or-later
```
Comment (Fabien-Chouteau) [inline index/pl/pla_util/pla_util-2.1.1.toml]: Hi @serock, we don't expect copyright headers in release manifests. The authors and licenses of the projects are already documented in the manifest.

**Review by Fabien-Chouteau — CHANGES_REQUESTED** [review]

File: index/pl/pla_util/pla_util-2.1.1.toml (hunk 1 above)
Comment (serock) [reply index/pl/pla_util/pla_util-2.1.1.toml]: Hi @Fabien-Chouteau, thank you for reviewing; the header has been removed.

File: index/pl/pla_util/pla_util-2.1.1.toml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+"linux" = true
+"..." = false
+
+[[depends-on]]
+libpcap = "^1.8.1"
+
+[configuration]
+generate_c = false
+
+[build-profiles]
```
Comment (Fabien-Chouteau) [inline index/pl/pla_util/pla_util-2.1.1.toml]: Sorry I missed that the first time, but you must not set build profiles in a release manifest.

@mosteo I am surprised that the CI didn't catch that.

**Review by Fabien-Chouteau — CHANGES_REQUESTED** [review]

File: index/pl/pla_util/pla_util-2.1.1.toml (hunk 2 above)
Comment (mosteo) [reply index/pl/pla_util/pla_util-2.1.1.toml]: I don't think we have checks for that, as it is conceivably desirable for non-library crates. Making a note of it.

Comment (serock) [reply index/pl/pla_util/pla_util-2.1.1.toml]: Hi @Fabien-Chouteau , thanks again for reviewing; I removed the build profiles.

**Review by Fabien-Chouteau — APPROVED** [review]

**Pull request merged**